installer/installer.go

```diff
@@ -208,6 +208,23 @@ func (i *Installer) GetArtifactName() string {
 	return i.ar.GetArtifactName()
 }
 
+// Returns a list of compatible devices
```

Comment:
Suggestion. Since these are already nicely commented. Why not add it to the go-doc?